src/commands/utility/info/emote.js

```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoEmoji extends Command {
+  constructor (client) {
+    super({
+      name: 'emoji',
+      parent: 'info',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_EMOJIS'] },
+      parameters: [{
+        type: 'emoji',
```

Comment:
It doesn't accept emoji IDs, like 445631099440463872 for example

Reply:
Yeah, because the emoji parameter never did.

Reply:
Oh^2